Use brand typography according to https://flur.ee/brand/

cap10morgan opened the pull request:
I noticed the admin UI was defaulting to browser serif font in a lot of places. Once @amandaloftis pointed me towards https://flur.ee/brand/ I decided to try applying that to the right places. Seemed like we had the "Cooper Hewitt for headings, buttons, etc." and "Open Sans for body text" backwards.

This also moves bootstrap.css out of the git repo and builds it on demand since it is generated from the LESS sources.

Before:
![fluree admin typography fix - before](https://user-images.githubusercontent.com/59748/112359112-8e945180-8c96-11eb-8d01-9be853512dfa.png)

After:
![fluree admin typography fix - after](https://user-images.githubusercontent.com/59748/112359145-95bb5f80-8c96-11eb-9b9b-3362d2ecde7c.png)

cap10morgan committed:
Add Cooper Hewitt font

cap10morgan committed:
Use Open Sans for body text, CH for headings, etc.

Following https://flur.ee/brand/

cap10morgan committed:
Ignore bootstrap.css and teach make how to generate it

amandaloftis commented:
Everything looks good, but can we update the menu headers (on the left) to be Cooper Hewitt Medium too? They're currently very thin. @cap10morgan

cap10morgan committed:
Remove Cooper Hewitt light font

Not sure why but this was making the sidebar use the lighter font weight (even though it wasn't requesting it AFAICT). 🤷🏼‍♂️

cap10morgan commented:
@amandaloftis Yeah, strange. It was using the lighter weight font for some reason. I just removed that font b/c nothing was explicitly requesting it AFAICT.

How's this look?

![CleanShot 2021-03-25 at 11 20 21@2x](https://user-images.githubusercontent.com/59748/112515826-3fb0ef80-8d5c-11eb-8222-f1b9464f9fb4.png)

jakep36 commented:
👍
